### PR TITLE
Build modern api using fastify

### PR DESCRIPTION
**Pull request by reverentgeek**
This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [X] A title that's approved by Dev Marketing
- [X] A URL that's approved by Dev Marketing
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2020-10-12-build-modern-api-using-fastify.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+For this tutorial, you will create a secure API that returns some information regarding employees. To authenticate users, you will use Okta as an authentication server which will produce a JSON Web Token (JWT) after authenticating the user. The client will then send that JWT as part of the request to the server which will handle the validation logic. You will use Okta's JWT verifier to quickly set up the authentication on your server.
+
+## Create your Okta application
+
```
Comment (dogeared) [inline _source/_posts/2020-10-12-build-modern-api-using-fastify.md:22]: May need some boilerplate here on setting up an okta org. Browse to developer.okta.com/signup, etc.

File: _source/_posts/2020-10-12-build-modern-api-using-fastify.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+For this tutorial, you will create a secure API that returns some information regarding employees. To authenticate users, you will use Okta as an authentication server which will produce a JSON Web Token (JWT) after authenticating the user. The client will then send that JWT as part of the request to the server which will handle the validation logic. You will use Okta's JWT verifier to quickly set up the authentication on your server.
+
```
Comment (dogeared) [inline _source/_posts/2020-10-12-build-modern-api-using-fastify.md:20]: Lot of "will" language, which weakens the point of view.

"For this tutorial, you create..." is stronger. Not a deal breaker, though. ;)

**Review by dogeared — APPROVED** [review]
Needs some Okta setup. Other than that, good to go!

File: _source/_posts/2020-10-12-build-modern-api-using-fastify.md (hunk 1 above)
Comment (reverentgeek) [reply _source/_posts/2020-10-12-build-modern-api-using-fastify.md:22]: I can't make these changes where I am, but I'll update them later tonight.

**Pull request merged**